ui.R

```diff
@@ -87,6 +87,73 @@ navbarPage("Power Implications for Reduced Sample Size",
                       )
                     )
            ),
+           
+           # ***************************************************
+           # GSD start
+           # ***************************************************
+           tabPanel("GSD",
+                    sidebarLayout(
+                      sidebarPanel(
+                        selectInput(inputId = "designSelect", label="Type of GSD", 
+                                    choices = c("Pocock" = "P", 
+                                                "O'Brien-Fleming"="OF"), 
+                                    selected="Pocock"),
+                        numericInput(
+                          inputId = "alphaGSD",
+                          label = "One-sided significance level",
+                          value = 0.025,
+                          min = 0.001,
+                          step = 0.001
+                        ),
+                        numericInput(
+                          inputId = "powerGSD",
+                          label = "Power (%)",
+                          value = 90,
+                          min = 1,
+                          step = 0.1
+                        ),
+                        bsTooltip("powerGSD",
+                                  "Power for statistical significance to detect a hypothesized design effect (>0)",

```

Comment:
Do you think it is clearer if we emphasize that this power is for a fixed sample test.? For example, "Power for statistical significance to detect a hypothesized design effect (>0) using the fixed sample test".